cluster/gce/manifests/kube-apiserver.manifest

```diff
@@ -32,17 +32,19 @@
     {{container_env}}
     "livenessProbe": {
       "httpGet": {
+        "scheme": "HTTPS",
         "host": "127.0.0.1",
-        "port": 8080,
+        "port": {{secure_port}},
```

Comment:
Does this default to 443 now? Do we allow user to switch back to 8080 and insecure?

Reply:
It's 443 in our [startup configuration script](https://github.com/kubernetes/kubernetes/blob/ac1557a8861b2c8fe50a2749ea45169a94bd4756/cluster/gce/gci/configure-helper.sh#L1595), but I made it {{secure_port}} here to stay consistent with the other instances in the manifest.

Reply:
I'm not sure there's a point in using the insecure port for the liveness or readiness probes

Reply:
I'm guessing that there was also intended to be an {{insecure_port}}, which never got hooked up: https://github.com/kubernetes/kubernetes/blob/ac1557a8861b2c8fe50a2749ea45169a94bd4756/cluster/gce/gci/configure-helper.sh#L1880-L1881

Reply:
SG, thanks for digging that up, I was just not clear on the expected behavior.

Reply:
BTW should we remove `sed -i -e "s@{{secure_port}}@8080@g" "${src_file}"`? That was a bit confusing :)

Reply:
yeah, it's a no-op at best. done.